metadata_backend/helpers/schemas/BP.bpstaining.xsd

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:bp="BP.common">
+  <xs:import namespace="BP.common" schemaLocation="BP.common.xsd"/>
+
+  <xs:complexType name="StainingSetType">
+    <xs:sequence minOccurs="1" maxOccurs="unbounded">
+      <xs:element name="STAINING" type="StainingType"/>
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:element name="STAINING_SET" type="StainingSetType">
+    <xs:annotation>
+      <xs:documentation>A container of staining (StainingType) objects. </xs:documentation>
+    </xs:annotation>
+  </xs:element>
+
+  <xs:element name="STAINING" type="StainingType"/>
+
+  <xs:complexType name="StainingType">
+    <xs:annotation>
+      <xs:documentation>
+        Describes stains used for a slide. Can be referencesd. Stains are either defined by procedure information or a list of stains.
```

Comment:
```suggestion
        Describes stains used for a slide. Can be referenced. Stains are either defined by procedure information or a list of stains.
```

not our typo i assume.

Reply:
Not our typo indeed. I found quite a few typos in the BP xml schemas when I would transfer the element descriptions to our json schemas because we currently check the spelling in json schemas but not in the XML schemas.